_layouts/404.html

```diff
@@ -15,29 +15,28 @@
 {% block content_main %}
     <div class="error_copy_container">
         <header class="error_header u-mt30">
-            <h1 class="superheader">Oops! 404 error.</h1>
-            <p class="short_desc">Something’s unplugged. We can’t find the page you 
-            	were looking for, but here are a number 
-            	of other ways you might try to get there.
+            <h1 class="superheader">Something’s disconnected.</h1>
+            <p class="short_desc">We couldn’t find the exact page you 
+            were looking for, but we have other information that may be useful.
             </p>
         </header>
         <aside class="error_action_container padded-container u-mt60 u-mb60">
             <div class="content-l content-l__large-gutters padded-container_body">
                 <div class="content-l_col content-l_col-1-2">
                     <p>
-                        Explore our other useful tools and resources.
+                        Explore our tools and resources.
                     </p>
                     <a href="/" class="btn btn__full btn__error-home">Visit homepage</a>
                 </div>
                 <div class="content-l_col content-l_col-1-2 content-l_col__before-divider">
                     <p>
-                        If the problem persists please let us know.
+                        If you’re sure this is the right place, let us know.
                     </p>
                     <a class="jump-link jump-link__right" href="/contact-us">
-                        Visit contact us page
+                        Contact us
```

Comment:
I thought about that. I think we send them there for a few reasons.

1. General inquiries is at the top of the contact page anyway.
2. We'd need to add another line (for both phone and email); not sure if this has layout implications.
3. If someone is looking for something specific, they're going to get an answer much faster if they find the contact info directly than if we have them go through the switchboard. This relies on them getting it right, of course, but by making the contact page we've made, we've also committed ourselves to trying to make it good at helping them picking a point of contact correctly.

Maybe not an ideal solution, but we can user-test it. I'll write some questions into the current round.